packages/formation-react/testing/karma.conf.js

```diff
@@ -15,6 +15,7 @@ module.exports = function karmaConfig(config) {
     frameworks: ['mocha'],
     webpack: { // kind of a copy of your webpack config
       devtool: 'inline-source-map', // just do inline source maps instead of the default
+      mode: 'none',
```

Comment:
some of our enzyme tests rely on querying a component's `displayName` using `node.find()`. `displayName` is only included in non production builds and webpack 4 defaults to production builds if no `mode` is set. `none` forces webpack to previous versions default behavior. 

Not exactly where what's doing this optimization- webpack or babel or react.